kernel/classes/ezpersistentobject.php

```diff
@@ -951,7 +951,7 @@ public static function handleRows( $rows, $class_name, $asObject )
         if ( is_array( $rows ) )
         {
             $db = eZDB::instance();
-            foreach( $rows as $row )
+            foreach( $rows as &$row )
```

Comment:
Do wee need to loop at all ? `replaceFieldsWithLongNames()` will already iterate on the fields passed as the 3rd argument. I'm thinking that replacing the whole `foreach` with `self::replaceFieldsWithLongNames( $db, $objectDefinition['fields'], $rows )` would just do the same thing.